.github/workflows/build-hw-level-zero.yml

```diff
@@ -0,0 +1,75 @@
+---
+name: Level Zero
+
+on: [push, pull_request]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+
+jobs:
```

Comment:
we should either remove it entirely and fold it into regular tests (I still think that the whole conda setup is unnecessary) or move the workflow to another file.

Reply:
I think there's still value in having this test and agree about the conda setup, I hope it's not broken while it's been disabled. Having it in a separate workflow makes sense.

Reply:
I moved it into its own workflow file: `examples-hw-level-zero.yml`